packages/multi-select-combo-box/theme/material/vaadin-multi-select-combo-box-chip-styles.js

```diff
@@ -23,12 +23,57 @@ const chip = css`
     cursor: default;
     white-space: nowrap;
     font-family: var(--material-font-family);
+    min-width: 0;
```

Comment:
This was missing from the Material theme causing long chips to not shrink properly.